bcgov/HOUS-permit-portal
# Feature/bphh 674/update multi upload error messages

## Description

Allow proper error messages for max size to show up, and better error handling when there is something wrong with teh backend for files.

## What type of PR is this? (check all applicable)

- [x ] 🍕 Feature


## Related Tickets & Documents

https://hous-bssb.atlassian.net/browse/BPHH-674


## Steps to QA

Set up a max size in MB.
Errors should show if you exceed that size.
Errors shoul dnot show if you uplaod less than that.
Disable your minio / bucket.
A proper error message should when you upload a file.
